src/components/common/TmSessionSignUp.vue

```diff
@@ -4,7 +4,14 @@
       <h2 class="session-title">
         Create a new address
       </h2>
-      <div v-if="!session.insecureMode && !session.mobile" class="session-main">
+      <div
+        v-if="
+          !session.insecureMode &&
+            !session.mobile &&
+            !(network.slice(-7) === `testnet`)
```

Comment:
🤢

Reply:
i think @iambeone made a helper function to check if it's a network - maybe we can use that or something?

Reply:
we have the `testnet` property for checking if this is a testnet

Reply:
> 🤢

xD a bit exaggerated no?

Reply:
yes, very exaggerated.